plugin/vscode/test/vscode-tests.js

```diff
@@ -52,9 +52,19 @@ for (let extension of [".js", ".mjs", ".cjs"]) {
 
       await pollAsync(async () => {
         let helloDiags = normalizeDiagnostics(helloURI);
+
         assert.deepStrictEqual(helloDiags, [
           {
-            code: "E0034",
+            code: {
+              target: {
+                scheme: "https",
+                authority: "quick-lint-js.com",
+                path: "/errors/E0034",
+                query: "",
+                fragment: "",
+              },
```

Comment:
Let's make `normalizeDiagnostics` convert the URI object into a string. This will let us simplify the assertion:
```suggestion
              target: "https://quick-lint-js.com/errors/E0034",
```